data/json/recipes/chem/mutagen.json

```
@@ -244,7 +245,7 @@
     "using": [ [ "mutagen_production_standard", 25 ] ],
     "components": [
       [ [ "mutagen", 1 ] ],
-      [ [ "meat_tainted", 3 ], [ "dry_meat_tainted", 3 ] ],
+      [ [ "meat_tainted", 3 ], [ "dry_meat_tainted", 3 ], [ "blood_tainted_concentrate", 9 ] ],
```

```suggestion
      [ [ "blood_tainted_concentrate", 9 ] ],
```
I would rather see only powder/fluids in making serums.